### PR TITLE
refactor(components): [checkbox] use useNamespace

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/checkbox/src/checkbox.vue [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
   <label
     :id="id"
-    class="el-checkbox"
     :class="[
-      checkboxSize ? 'el-checkbox--' + checkboxSize : '',
-      { 'is-disabled': isDisabled },
-      { 'is-bordered': border },
-      { 'is-checked': isChecked },
+      ns.b(),
+      checkboxSize ? ns.m(checkboxSize) : '',
```
Comment (sxzz) [inline packages/components/checkbox/src/checkbox.vue:6]:
```suggestion
      ns.m(checkboxSize),
```
If checkboxSize is falsey, ns.m will return an empty string.

File: packages/components/checkbox/src/checkbox-button.vue [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <template>
   <label
-    class="el-checkbox-button"
     :class="[
-      size ? 'el-checkbox-button--' + size : '',
-      { 'is-disabled': isDisabled },
-      { 'is-checked': isChecked },
-      { 'is-focus': focus },
+      ns.b('button'),
+      size ? ns.bm('button', size) : '',
```
Comment (sxzz) [inline packages/components/checkbox/src/checkbox-button.vue:5]: Same here

File: packages/components/checkbox/src/checkbox.vue (hunk 1 above)
Comment (btea) [reply packages/components/checkbox/src/checkbox.vue:6]: ok

File: packages/components/checkbox/src/checkbox-button.vue (hunk 2 above)
Comment (btea) [reply packages/components/checkbox/src/checkbox-button.vue]: Updated.

**Review by sxzz — APPROVED** [review]

**Pull request merged**